src/Ray.php

```diff
@@ -264,10 +264,37 @@ public function className(object $object): self
         return $this->send(get_class($object));
     }
 
+    public function phpinfo(string ...$properties): self
+    {
+        $default = [
+            'PHP version' => phpversion(),
+            'Timezone' => ini_get('date.timezone'),
+            'Charset' => ini_get('default_charset'),
+            'Memory limit' => ini_get('memory_limit'),
+            'Max file upload size' => ini_get('max_file_uploads'),
+            'Max post size' => ini_get('post_max_size'),
+            'Hostname' => php_uname('n'),
+            'PHP ini file' => php_ini_loaded_file(),
+            "PHP scanned ini file" => explode(',', str_replace(PHP_EOL, '', php_ini_scanned_files() ?? '')),
+            'Extensions' => implode(', ', get_loaded_extensions()),
+        ];
+
+        $properties = array_flip($properties);
+
+        foreach ($properties as $property => $value) {
+            $properties[$property] = ini_get($property);
+        }
+
+        return $this->sendCustom(
+            json_encode(array_merge($properties, $default)),
```

Comment:
It would be nice if we could display this as a table in Ray, like how the query timings are displayed

![image](https://user-images.githubusercontent.com/3626559/105500578-59ce4300-5cc3-11eb-82ab-53e70780e105.png)

Reply:
Maybe a `ray()->table($array)` method you could pass an array in?

Reply:
Yeah, already working on this. But it was a long week 😄 so didn't have time to look into it. Will do it next week